### PR TITLE
WIP: login portal dashboard 

### PR DESCRIPTION
**Pull request by bradewalder**
This is still a work in progress.

Functionality is yet to be added, I have created a basic login portal and dashboard (WIP) based around Ojasvi and I's original Figma designs.

### CODE REVIEW

**Review by MorgaineBarter — DISMISSED** [review]
thank you for sharing this on Friday in the meeting. looks good

**Review by MorgaineBarter — DISMISSED** [review]

**Review by MorgaineBarter — DISMISSED** [review]

**Comment by doubleedesign** [comment]
I'll pull and run this code locally ASAP to give more detailed feedback if needed and help get this through, but initial feedback is just to shuffle things around so that each component is in its own folder (inside `src/components`) with its own styling. Some relevant docs published earlier this week:
- [Creating components](https://redback-operations.github.io/redback-documentation/docs/web-mobile-app-dev/frontend/new-components/) (for the expected directory structure and naming conventions)
- [Using Styled Components](https://redback-operations.github.io/redback-documentation/docs/web-mobile-app-dev/frontend/styled-components/)
- [Submitting code](https://redback-operations.github.io/redback-documentation/docs/web-mobile-app-dev/frontend/submitting-work) (more about the linting/structure checks that are now running automatically on PRs to help us keep things consistent)

I'm happy to lend a hand with this if I get time in the next week (but can't promise what time I'll have unfortunately, so don't wait on me).

**Comment by doubleedesign** [comment]
Have set this up locally; loving the animation to switch between login and signup! There could be a few things here to tweak next trimester and maybe some to bring into Redback UI (also next trimester), but overall once we get it passing the checks I'm happy with this as an MVP to be merged.

**Comment by ben-AI-cybersec** [comment]
@doubleedesign could you please give us the file naming conventions? The linting errors aren't super clear.

**Comment by doubleedesign** [comment]
I've just pushed up a change that corrects the directory structure and naming conventions. I can see that in the future we may want to allow component folders within component folders, but I haven't had a change to tinker with that as yet. So for now at least, the basic formula is:

- routes/
  - RouteName/
    - RouteName.tsx
    - RouteName.style.ts
    - RouteName.test.tsx
- components/
  - ComponentName/
    - ComponentName.tsx
    - ComponentName.style.ts
    - ComponentName.test.tsx

Noting that the file names match the folder names, and all are in `PascalCase` (this is the general React convention). 

I suspect there will also be a `hooks` folder in future, when we come across the need for custom hooks (which will follow the React convention of `camelCase` starting with `use`).

**Review by bradewalder — COMMENTED** [review]
Thank you! Looks great!

**Comment by doubleedesign** [comment]
No worries @bradewalder! Just the CSS refactoring left now I think, which as discussed on Teams I'm happy to help with ASAP. There's quite a bit here that could potentially be adapted and built upon to be used in other parts of the project and in the other teams' projects too, great work!

**Comment by doubleedesign** [comment]
As discussed on Teams, I have raised a PR to allow CSS modules in addition to Styled Components for now, as it will help us to finalise this PR and we can easily convert them later when Redback UI is integrated and we are thus more ready for the dynamic stuff that Styled Components makes available.

**Comment by bradewalder** [comment]
Thank you @doubleedesign! I will keep that in mind for the future. Appreciate your insights, they are very valuable.

File: CODEOWNERS [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # These owners will be the default owners for everything in
 # the repo. Unless a later match takes precedence,
-# @global-owner1 and @global-owner2 will be requested for
+# @bradewalder will be requested for
 # review when someone opens a pull request.
-*	@Redback-Operations/wearables-for-athletes @Redback-Operations/tutors 
+* @bradewalder @Redback-Operations/wearables-for-athletes @Redback-Operations/tutors
```
Comment (doubleedesign) [inline CODEOWNERS]: @bradewalder Please remove yourself from the codeowners - I know you're a senior student so you probably shouldn't be an approver once you've finished capstone ;)

**Review by doubleedesign — CHANGES_REQUESTED** [review]

**Review by ben-AI-cybersec — APPROVED** [review]
Moving T2 work from entire team into main repo so we can work through checking it.

**Pull request merged**